common/txmgr/confirmer.go

```diff
@@ -10,6 +10,8 @@ import (
 	"sync"
 	"time"
 
+	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
```

Comment:
evm specific code should not be importer into common pkg. Use generic type (the same defined used in Confirmer)